### PR TITLE
[CoreBundle] Fixing resource search.

### PR DESCRIPTION
**Pull request by ngodfraind**
We dropped the support for directory shortcuts a long time ago.
This code was buggy and now is removed.

### CODE REVIEW

File: main/core/Repository/ResourceNodeRepository.php [hunk 1]
```diff
@@ -284,21 +284,6 @@ public function findAncestors(ResourceNode $resource)
     public function findByCriteria(array $criteria, array $roles = null, $isRecursive = false)
```
Comment (stefk) [inline main/core/Repository/ResourceNodeRepository.php]: The `$isRecursive` parameter should be removed and the docblock updated.

**Comment by LaurentGruber** [comment]
LGTM

**Comment by Elorfin** [comment]
LGTM

**Comment by Elorfin** [comment]
LGTM

**Pull request merged**